src/quo2/components/dividers/strength_divider/component_spec.cljs

```diff
@@ -0,0 +1,18 @@
+(ns quo2.components.dividers.strength-divider.component-spec
+  (:require [quo2.components.dividers.strength-divider.view :as strength-divider]
+            [test-helpers.component :as h]))
+
+(h/describe "select-profile component"
+  (h/test "render component"
+    (h/render [strength-divider/view {:status :okay}])
+    (-> (h/expect (h/get-by-label-text :strength-divider))
+        (.toBeTruthy)))
+  (h/test "render component with correct status"
+    (h/render [strength-divider/view {:status :strong}])
+    (-> (h/expect (h/get-by-text "tx:strength-divider-status-strong"))
+        (.toBeTruthy)))
```

Comment:
we can also add another helper to the utilities file.
utilities
```
(defn get-by-translation-text [keyword]
  (str "tx:" keyword))
```
test file
```
(h/get-by-translation-text :strength-divider-status-strong))
        (.toBeTruthy)
```       
No need to add in this pr if it's too much, but probably we will add something like this in a follow up 👍

Reply:
I will add it to this pr.

Reply:
I have added the `get-by-translation-text` helper function and tests for all statuses.